### DISCUSSION TITLE
Hey all, please tell me the country/countries you would like to visit in the future?

### DISCUSSION THREAD

**Discussion by veekay7** [NAIST-SE/AttractingContributors#234] [Ideas]
Hey all, 

I'd really love to know what country/countries do you all want to visit in the future. Please fork the repo and let's share our dreams. ^^

https://github.com/VantelinLuca/AttractingContributors-WishCountry

**Comment by TakumiHiraide** [comment]
Added, please help me!!
https://github.com/TakumiHiraide/My-favorite-sports

**Comment by t-matsumoto4758** [comment]
Added!
please check and help me !
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by xincanfeng** [comment]
added!
please tell me your favorite color!
https://github.com/FengXincan/AttractingContributors-color

**Comment by ellielo** [comment]
I am very excited to visit Australia and New Zealand in the future. Hope they will be open for the tourist in the nearest future. I am prepared, just ordered myself lots of masks and sanitizers from [online pharmacy](https://www.canadadrugsdirect.com/online-pharmacy) :) But of course would be nice if everything goes back to normal one day and we don't need all those anymore.
